swig/matlab/helicsEndFederate.m

```diff
@@ -0,0 +1,19 @@
+function helicsEndFederate(fed_info)
```

Comment:
wondering if helicsCloseFederate might be a better name?

Reply:
Maybe `helicsDestroyFederate` that would be symmetrical with the the various "create" actions that made it in the first place.

Incidentally, next code change (after move to examples repo) I want to make to this is to replace the programmatic build up of the federate with creating it from JSON, which seems like the paradigm we want to steer folks toward.

Overall (and after moving this code to examples) I could see there being 3 simple examples, many/all of which we might opt to duplicate for other languages over time:
1. Simple Value Federate: we have this one: `pisender/pireceiver`
2. Simple Message Federate: perhaps `hellosender`/`helloreceiver` which might also include the use of a delay filter.
3. Modestly more complex combo federate, that in particular shows the use of a value and message stream for similar data (such as physical voltage vs. measured voltage). Ideally this might also show the use of a built-in player for one of the data streams.

All of these would use helicsDestroyFederate--likely multiple times during error checks--and all would want to run helicsDestroyFederate. It is not until we get to more advanced example that we'd need to skip closing the library. Such as:
* Future example with multiple federates sharing the same broker, built up programmatically in a loop

Reply:
I can add those functions in the C interface for the 2.0 alpha release.  There would probably also be a corresponding helicsDestroyCore,  helicsDestoryBroker as well.  

You can also use TOML now if you so desire and that is easier.

Reply:
Renamed